packages/date-picker/test/overlay-content.test.js

```diff
@@ -455,7 +455,7 @@ describe('overlay', () => {
       it('should scroll when the month is below the visible area', () => {
         const position = monthScroller.position;
         overlay.revealDate(new Date(2021, 3, 1), false);
-        expect(monthScroller.position).to.equal(position + 0.6 /* The bottom 10% offset is ensured by JS */);
+        expect(monthScroller.position).to.be.closeTo(position + 0.6, 0.1 /* The bottom 10% offset is ensured by JS */);
```

Comment:
I'm not very happy about this test now - it has something to do with the offset affected by base styles and the error was like `expected -51.45454545454545 to equal -51.4`. But we can probably improve this later.